Functions.Templates/Templates/IoTHubServiceBusQueueTrigger-CSharp/metadata.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "IoT Hub (ServiceBusQueue)",
```

Comment:
Spaces? (Service Bus Queue)?

Reply:
The names were the ones I got from @mattchenderson so maybe we should go through these with him?

Reply:
If we're spacing everything else, it would make sense to space these. @mattchenderson is the right person to answer.